src/main/java/org/gridsuite/explore/server/services/FilterService.java

```diff
@@ -6,139 +6,94 @@
 
-import static org.gridsuite.explore.server.ExploreException.Type.FILTER_NOT_FOUND;
-
 /**
  * @author Etienne Homer <etienne.homer at rte-france.com>
  */
 @Service
 public class FilterService implements IDirectoryElementsService {
-    private static final String ROOT_CATEGORY_REACTOR = "reactor.";
-
     private static final String FILTER_SERVER_API_VERSION = "v1";
 
     private static final String DELIMITER = "/";
 
-    private final WebClient webClient;
     private String filterServerBaseUri;
 
+    private final RestTemplate restTemplate;
+
     @Autowired
-    public FilterService(@Value("${backing-services.filter-server.base-uri:http://filter-server/}") String filterServerBaseUri,
-                         WebClient.Builder webClientBuilder) {
+    public FilterService(@Value("${backing-services.filter-server.base-uri:http://filter-server/}") String filterServerBaseUri, RestTemplate restTemplate) {
         this.filterServerBaseUri = filterServerBaseUri;
-        this.webClient = webClientBuilder.build();
+        this.restTemplate = restTemplate;
     }
 
     public void setFilterServerBaseUri(String filterServerBaseUri) {
         this.filterServerBaseUri = filterServerBaseUri;
     }
 
-    public Mono<Void> replaceFilterWithScript(UUID id) {
+    public void replaceFilterWithScript(UUID id) {
         String path = UriComponentsBuilder.fromPath(DELIMITER + FILTER_SERVER_API_VERSION + "/filters/{id}/replace-with-script")
                 .buildAndExpand(id)
                 .toUriString();
-
-        return webClient.put()
-                .uri(filterServerBaseUri + path)
-                .retrieve()
-                .onStatus(httpStatus -> httpStatus == HttpStatus.NOT_FOUND, clientResponse -> Mono.error(new ExploreException(FILTER_NOT_FOUND)))
-                .bodyToMono(Void.class)
-                .publishOn(Schedulers.boundedElastic())
-                .log(ROOT_CATEGORY_REACTOR, Level.FINE);
+        restTemplate.exchange(filterServerBaseUri + path, HttpMethod.PUT, null, Void.class);
     }
 
-    public Mono<Void> insertNewScriptFromFilter(UUID id, UUID newId) {
+    public void insertNewScriptFromFilter(UUID id, UUID newId) {
         String path = UriComponentsBuilder.fromPath(DELIMITER + FILTER_SERVER_API_VERSION + "/filters/{id}/new-script?newId={newId}")
-            .buildAndExpand(id, newId)
-            .toUriString();
-
-        return webClient.post()
-                .uri(filterServerBaseUri + path)
-                .retrieve()
-                .onStatus(httpStatus -> httpStatus == HttpStatus.NOT_FOUND, clientResponse -> Mono.error(new ExploreException(FILTER_NOT_FOUND)))
-                .bodyToMono(Void.class)
-                .publishOn(Schedulers.boundedElastic())
-                .log(ROOT_CATEGORY_REACTOR, Level.FINE);
+                .buildAndExpand(id, newId)
+                .toUriString();
+        restTemplate.exchange(filterServerBaseUri + path, HttpMethod.POST, null, Void.class);
```

Comment:
keep the following behaviour :
.onStatus(httpStatus -> httpStatus == HttpStatus.NOT_FOUND, clientResponse -> Mono.error(new ExploreException(FILTER_NOT_FOUND)))